Sources/Purchasing/Purchases.swift

```diff
@@ -91,9 +91,12 @@ public typealias StartPurchaseBlock = (@escaping PurchaseCompletedBlock) -> Void
     private let operationDispatcher: OperationDispatcher
 
     /**
-     * Enable automatic collection of Apple Search Ads attribution. Defaults to `false`.
+     * Enable automatic collection of AdServices attribution token. Defaults to `false`.
+     *
+     * Should match OS availability in https://developer.apple.com/documentation/ad_services
      */
-    @objc public static var automaticAppleSearchAdsAttributionCollection: Bool = false
+    @available(iOS 14.3, macOS 11.1, macCatalyst 14.3, *)
```

Comment:
This is being implicitly marked as available for _any_ version of `tvOS` and `watchOS`. You need to add this:
```swift
@available(tvOS, unavailable)
@available(watchOS, unavailable)
```